### ISSUE TITLE
Spawning holograms in ChunkLoadEvent -> wrong line location sent to player

### ISSUE THREAD

**Issue by PierreSchwang** [DecentSoftware-eu/DecentHolograms#105]
### Just making sure

- [X] I made sure, this bug is not already known or reported.
- [X] I believe, that this bug is fixable.

### Reproduction

Spawning Holograms using the API in the ChunkLoadEvent (even with a delay) fails to position the lines. The Location returned by `HologramLine#getLocation()` is correct somehow.

Code for spawning the Hologram: https://github.com/IntellectualSites/HoloPlots/blob/feat/decentHolograms/src/main/java/com/plotsquared/holoplots/provider/impl/DecentHologramsProvider.java#L31

Triggered by the ChunkLoadEvent: https://github.com/IntellectualSites/HoloPlots/blob/feat/decentHolograms/src/main/java/com/plotsquared/holoplots/listener/ChunkListener.java#L28

When debugging the locations in the createHologram method, the locations are correct as I said:
```
[20:38:25 INFO]: [HoloPlots] [STDOUT] Line at Location{world=CraftWorld{name=test},x=-192.5,y=65.45,z=-95.5,pitch=0.0,yaw=0.0}
[20:38:25 INFO]: [HoloPlots] [STDOUT] Line at Location{world=CraftWorld{name=test},x=-192.5,y=64.9,z=-95.5,pitch=0.0,yaw=0.0}
[20:38:25 INFO]: [HoloPlots] [STDOUT] Line at Location{world=CraftWorld{name=test},x=-192.5,y=64.60000000000001,z=-95.5,pitch=0.0,yaw=0.0}
[20:38:25 INFO]: [HoloPlots] [STDOUT] Line at Location{world=CraftWorld{name=test},x=-192.5,y=64.30000000000001,z=-95.5,pitch=0.0,yaw=0.0}
[20:38:25 INFO]: [HoloPlots] [STDOUT] Line at Location{world=CraftWorld{name=test},x=-192.5,y=64.00000000000001,z=-95.5,pitch=0.0,yaw=0.0}
[20:38:25 INFO]: [HoloPlots] [STDOUT] Created HoloPlots__test_null_-3;-1
```

Example results when spawned in ChunkLoadEvent: 
![grafik](https://user-images.githubusercontent.com/27054324/211929077-9098d483-621b-44bf-bc9a-845a5e88f2f6.png)
![grafik](https://user-images.githubusercontent.com/27054324/211929092-af0cd90e-f114-4944-8ee6-73153301deb0.png)

And when spawned because of a plot modification (chunk is loaded and player is nearby):
![grafik](https://user-images.githubusercontent.com/27054324/211929145-75880aab-6c6b-4d3e-9ed2-c89cfb51183e.png)

### Server Version

git-Paper-310 (MC 1.19.3)

### Client Version

1.19.3

### Plugin Version

2.7.9

### Log

https://paste.gg/p/anonymous/7075ff6ccfd3468f98733f0457bedf31

**Comment by d0by1** [comment]
Hello, 

does this happen every time or only in newly created chunks? The event you are using has a method to check if the chunk is new or not. I *think* that when a new chunk is still being populated, the entities might get misplaced by the client.

**Comment by PierreSchwang** [comment]
Hi! The chunks are / were already populated.

Haven't worked on it for a while tbh - iirc those were as well be spawn chunks. WIll have to try again and provide better details

**Comment by d0by1** [comment]
How far does the player have to be from the holograms for them to spawn?

**Comment by PierreSchwang** [comment]
I haven't changed the spawn behaviour at all. I spawn / register / create them not based on players nearby, but on all plot updates or chunk loads.
So the player distance is set by DH itself

**Comment by d0by1** [comment]
I made some changes in v2.7.10, I'm not sure if I solved your issue. Let me know if anything changed.

**Comment by PierreSchwang** [comment]
Heya, tested really quick - The issue still persists in 2.7.10

**Comment by TheJoshue** [comment]
Will this be solved any soon? Just asking.
More than half a year passed.

**Comment by zestian56** [comment]
This could be a good improvement

**Comment by Mathildeuh** [comment]
It is patch ?

**Comment by d0by1** [comment]
Hello, this issue has been fixed in 2.8.5.
